.gitignore

```diff
@@ -40,3 +40,4 @@ docker/fluxy-dumbconf.priv
 test/profiles
 test/bin/kubectl
 test/bin/helm
+.idea
```

Comment:
This belongs in your personal `.gitignore` as it is environment specific (e.g. `~/.gitignore`).

Reply:
yeah u'r right... sorry 😩